packages/ketcher-core/src/application/render/renderers/BaseMonomerRenderer.ts

```diff
@@ -278,6 +279,13 @@ export abstract class BaseMonomerRenderer extends BaseRenderer {
     }
   }
 
+  public moveSelection(offset: Vec2) {
+    assert(this.rootElement);
+    this.monomer.moveRelative(offset);
```

Comment:
Please move this row to DrawingEntitiesManager. Renderers should not change the model. We created DrawingEntitiesManager for this purpose.